password-manager-start/ui.py

```diff
@@ -13,9 +13,12 @@
 # Buttons Text
 GEN_PASS_BTN = 'Generate Password'
 ADD_BTN = 'ADD'
+SEARCH_BTN = 'Search'
```

Comment:
Same issue as other comment

Reply:
Please ignore this comment I will take care in future